client/src/components/pages/Playlist.js

```diff
@@ -1,41 +1,62 @@
-import React from 'react';
-import { Component } from 'react';
+import React, { Component } from 'react';
 import API from '../../utils/API';
-import { AppNavbar, VideoList } from '../index';
-import { Jumbotron } from 'react-bootstrap';
+import { VideoDetail, VideoList, AppNavbar } from '../index';
 
 class Playlist extends Component {
 	constructor(props) {
 		super(props);
 		this.state = {
-			videos: []
+			videos: [],
+			selectedVideo: ''
 		};
 	}
-	componentDidMount() {
-		this.handleGetVideos();
-	}
+	onVideoSelect = video => {
+		this.setState({
+			selectedVideo: video
+		});
+	};
 
 	handleGetVideos = () => {
 		API.getSavedVideos()
 			.then(res => {
 				this.setState({
-					videos: res.data
+					videos: res.data,
+					selectedVideo: res.data[0]
 				});
 				console.log(res.data);
 			})
 			.catch(err => console.log(err));
 	};
+
+	deleteVideo = (event, id) => {
+		event.preventDefault();
+		API.deleteVideo(id)
+			.then(res => this.handleGetVideos())
```

Comment:
Similar to my comments for handleSave in home component. Remove event and id from parameter. deleteVideo function doesn’t need the event and Id is not coming into the function on click. You already have access to the selectedVideo in playlist so the id will be selectedVideo._id. 

Also I think you should start using async function. Callback are not really used in modern JavaScript.

Eg.
deleteVideo = async() =>{
     try{
         await API.deleteVideo(selectedVideo._id);
this.handleGetVideos();
}catch(error){
alert(“display your message here if anything went wrong while deleting ”);
}
}

See how async function is beautiful...😉